logger/logging_config.py

```diff
@@ -0,0 +1,18 @@
+import json
+from logging.config import dictConfig  # type: ignore
+
+done_setup = False
+
+
+def setup_logging():
+    global done_setup
+
+    if not done_setup:
+        try:
+            logging_config_file_path = 'logger/logging_config.json'
```

Comment:
My preference would be to just use the string directly in the line below and avoid the creation of an extra variable